src/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
React should not be imported if we don't use it

Reply:
removed unused import